Formula/automake.rb

```diff
@@ -15,20 +16,10 @@ class Automake < Formula
 
   depends_on "autoconf"
 
-  # Download more up-to-date config scripts.
-  resource "config" do
-    url "https://git.savannah.gnu.org/cgit/config.git/snapshot/config-0b5188819ba6091770064adf26360b204113317e.tar.gz"
-    sha256 "3dfb73df7d073129350b6896d62cabb6a70f479d3951f00144b408ba087bdbe8"
-    version "2020-08-17"
-  end
-
   def install
     on_macos do
       ENV["PERL"] = "/usr/bin/perl"
-    end
-
-    resource("config").stage do
-      cp Dir["config.*"], buildpath/"lib"
+      ENV.delete "M4"
```

Comment:
Why are we removing `M4` here?

Reply:
The build fails if we don't because of superenv.